packages/plugin-analytics/src/utils/PlausibleAnalytics.ts

```diff
@@ -0,0 +1,100 @@
+import { logger } from 'robo.js'
+import { BaseEngine, EventOptions, ViewOptions } from './analytics'
+
+export class PlausibleAnalytics extends BaseEngine {
+	private _PLAUSIBLE_DOMAIN = process.env.PLAUSIBLE_DOMAIN
+
+	public async view(page: string, options: ViewOptions): Promise<void> {
+		const temp = {
```

Comment:
I think this would look nicer as destructured like our other code. 

```ts
const {
  domain = this._PLAUSIBLE_DOMAIN,
  name = `https://${this._PLAUSIBLE_DOMAIN}${page}`,
  url
} = options ?? {}
```